bazel/README.md

```diff
@@ -97,34 +106,49 @@ for how to update or override dependencies.
     version of `ar` on the PATH, so if you run into issues building third party code like luajit
     consider uninstalling binutils.
 
-    On Windows, additional dependencies are required:
-
-    Install the [MSYS2 shell](https://msys2.github.io/) and install the `diffutils`, `patch`,
-    `unzip`, and `zip` packages using `pacman`. Set the `BAZEL_SH` environment variable to the path
-    of the installed MSYS2 `bash.exe` executable. Setting the `MSYS2_ARG_CONV_EXCL` environment
-    variable to a value of `*` is often advisable to ensure argument parsing in the MSYS2 shell
-    behaves as expected.
-
-    `Git` is required. The version installable via MSYS2 is sufficient.
+    ### Windows
+    On Windows, you'll need to install several dependencies manually.
 
-    Install the Windows-native [python3](https://www.python.org/downloads/), the POSIX flavor
+    [python3](https://www.python.org/downloads/): Specifically, the Windows-native flavor. The POSIX flavor
```

Comment:
I run into an issue with the store installation of python (https://github.com/msys2/MSYS2-packages/issues/1943).

Should we mention that python should be installed from the website? The reason I am bringing this up is if you type python on cmd/powershell the MSFT store pops up.

Reply:
I can add an explicit warning for the store app like we have for MSYS2 flavor